### PR TITLE
Reorganisation

### PR DESCRIPTION
**Pull request by enolfc**
# Summary 

Move things around to match the discussion on how we should organise the docs

----

**Related issue :**

### CODE REVIEW

**Comment by enolfc** [comment]
Missing the CODEOWNERS... getting there

**Review by gwarf — CHANGES_REQUESTED** [review]
Great!
From a first check:
* remove the GitHub section, it should go under About as contributing, cf the doc
* split HTC over multiple pages

**Comment by gwarf** [comment]
Maybe also better to remove the generic text from docsy like on http://localhost:1313/users/

**Comment by enolfc** [comment]
> Maybe also better to remove the generic text from docsy like on http://localhost:1313/users/ or http://localhost:1313/users/getting-support/

but still leave the placeholder?

**Comment by gwarf** [comment]
Either add some initial short text (better) or keep the placeholder (acceptable I would say)

**Review by andrea-manzi — APPROVED** [review]
All good from my side!

**Review by gwarf — DISMISSED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -24,6 +24,9 @@ content/*/*/datahub/ @andrea-manzi
 # Data Transfer
 content/*/*/data-transfer/ @andrea-manzi
 
+# Data Transfer
```
Comment (andrea-manzi) [inline CODEOWNERS:27]:
```suggestion
# Online Storage
```

**Review by gwarf — APPROVED** [review]
LGTM, thanks!

**Review by andrea-manzi — APPROVED** [review]
Good for me

**Pull request merged**